wp_com/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "wp_com"
+version = "0.1.0"
+edition = "2021"
```

Comment:
We've updated the edition of all other packages to `2024` recently.

Reply:
🙈 Addressed in https://github.com/Automattic/wordpress-rs/pull/586/commits/d7e68b148b2da254770c0d271940e1e10ae394d5